### ISSUE TITLE
Pygame 2.5.0 - 2.6.1 crahes on android with  NotImplementedError: display module not available (ImportError: dlopen failed: cannot locate symbol "alphablit_alpha_sse2_argb_surf_alpha" referenced by "/data/data/org.test.myapp/files/app/_python_bundle/site-packages/pygame/surface.so"...)

### ISSUE THREAD

**Issue by kengoon** [pygame/pygame#4394]
NotImplementedError: display module not available (ImportError: dlopen failed: cannot locate symbol "alphablit_alpha_sse2_argb_surf_alpha" referenced by "/data/data/org.test.myapp/files/app/_python_bundle/site-packages/pygame/surface.so"...)

**Comment by LuckyIntel** [comment]
Hello!As far as i know Android devices don't have SIMD, instead they have an Advanced SIMD named Neon.

SOURCE : https://developer.android.com/ndk/guides/cpu-arm-neon (Official Android about NEON SIMD)

You might have to compile the source with android SIMD support instead with GNU GCC.I don't see anything else that can cause this.You can try this while someone else tries to assist this issue.

Happy coding!

**Comment by kengoon** [comment]
@LuckyIntel I don't really know much but adding this `surface src_c/simd_blitters_sse2.c src_c/simd_blitters_avx2.c src_c/surface.c src_c/alphablit.c src_c/surface_fill.c $(SDL) $(DEBUG)` to Setup.Android.... fixed the crashing and it works perfectly.

**Comment by LuckyIntel** [comment]
Hmmm...But SIMD doesn't even work on Android without using flags.This can cause optimization problems.

**Comment by kengoon** [comment]
I don't know much, if there's a better way to get it done, I'm open to trying out new things. Currently I don't know anything about NEON and how to go about it. All I just want is for the code to work just fine.

**Comment by LuckyIntel** [comment]
Glad the code works now but if the code starts to run very slow you can consider using Neon SIMD, happy coding!

**Comment by kengoon** [comment]
@LuckyIntel yes it's quite slow the difference from that of desktop is very noticable. How do I use NEON?

**Comment by LuckyIntel** [comment]
To be honest with python, i exactly have no idea.I am kinda new to python and pygame.

For C++ : https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html#index-mfpu-1

In the first reply i made to your issue, the github issues i referred to you seems to use neon in pygame with-in the python.You can check them.

I might recommend you to check this : https://github.com/pygame/pygame/issues/1723

**Comment by LuckyIntel** [comment]
Didn't see the edit sorry.Yes ``` -mfpu=neon ``` is the default neon compilation flag for GNU GCC (C and C++ compiler)

**Comment by kengoon** [comment]
@LuckyIntel alright thanks man. I already checked out the 2 issues you dropped and did some extra digging online. Seem like I have to add `CFLAGS="-mfpu=neon"`. I will try that out again with `buildozer` and see if that works

**Comment by LuckyIntel** [comment]
I am glad i was able to help!If the issue gets fixed you can close the issue, if the issue persist you can always create a new issue or continue this one!Good luck.

**Comment by kengoon** [comment]
@LuckyIntel didn't work throws the same error. I will go with the current fix though and leave this issue open
